packages/paste-website/src/pages/utilities/grid/index.mdx

```diff
@@ -0,0 +1,533 @@
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Grid to align content to the right or left side of a page layout.">
+    <Box padding="space60">
+      <Grid>
+        <Column span={4} offset={8}>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t use Grid with only one column of content. This causes unnecessary elements. Box is what you’re looking for here.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column>
+          <Paragraph>Unnecessary HTML elements. Use <SiteLink to="/primitives/box">Box</SiteLink> instead.</Paragraph>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use Grid to lay out UI elements into consistent columns.">
+    <Box padding="space60">
+      <Grid gutter="space30">
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+      <Box marginTop="space50">
+        <Grid gutter="space30">
+          <Column>
+            <Card>
+              <Paragraph>Card content</Paragraph>
+            </Card>
+          </Column>
+          <Column>
+            <Card>
+              <Paragraph>Card content</Paragraph>
+            </Card>
+          </Column>
+        </Grid>
+      </Box>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don’t let columns grow too small. Gutters should never be wider than columns.">
+    <Box padding="space60">
+      <Grid gutter="space120">
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+        <Column>
+          <Card>
+            <Paragraph>Card content</Paragraph>
+          </Card>
+        </Column>
+      </Grid>
+    </Box>
+  </Dont>
+</DoDont>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/grid
+```
+
+#### Usage
+
+```jsx
+import {Grid, Column} from '@twilio-paste/grid';
+
+<Grid>
+  <Column>
+    Foo
+  </Column>
+</Grid>
+```
+
+#### Props
```

Comment:
Is it worth splitting this props table into 2, I’m unsure which props are for which component at this stage.

You could have 

Grid props

Table 

Column props

Table